cluster-autoscaler/estimator/estimator.go

```diff
@@ -63,7 +63,7 @@ func NewEstimatorBuilder(name string, limiter EstimationLimiter, orderer Estimat
 // scale-up is limited by external factors.
 type EstimationLimiter interface {
 	// StartEstimation is called at the start of estimation.
-	StartEstimation([]*apiv1.Pod, cloudprovider.NodeGroup)
+	StartEstimation([]*apiv1.Pod, cloudprovider.NodeGroup, []EstimationLimit)
```

Comment:
So we used to have a generic `EstimationLimiter` interface with threshold-based implementation, but now we're also introducing another `EstimationLimit` interface that returns thresholds? It seems the new interface is really only compatible with `thresholdBasedEstimationLimiter` - or am I missing something?